packages/table/src/layers/focusCell.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+export interface IFocusedCellCoordinates {
```

Comment:
cool, then I would recommend extending `ICellCoordinates` with an empty interface and leaving a comment explaining that more things are coming